src/routes/events.js

```diff
@@ -59,13 +59,13 @@ router.post('/add', auth, async (req, res) => {
 		const user = await User.findOne({ uid: req.user.uid }).select('-password');
 		console.log('user:', user);
 		console.log('uid:', req.user.uid);
-		if (user.role != 'admin') res.status(401).json({ msg: 'Not authorized' });
+		if (user.role != 'admin') res.status(401).json({ code: 'EVENTS_ADD_UNAUTHORISED', msg: 'Not authorized' });
 		else {
 			let eventCreation = await Events.create(event);
 			if (eventCreation) {
-				return res.status(201).json({ msg: 'Event Added Successfully' });
+				return res.status(201).json({ code: 'EVENTS_ADD_DONE', msg: 'Event Added Successfully' });
 			} else {
-				return res.status(400).json({ msg: 'Failed: Add Event Operation' });
+				return res.status(400).json({ code: 'EVENTS_ADD_FAILED', msg: 'Failed: Add Event Operation' });
```

Comment:
It'll be `EVEN`T not `EVENTS` as we're doing everything on a single event only!

Reply:
Done.
Ref: 6a2ddc7ac52071efdfeddcaf94fc721f8e70d1ff